cmd/webhook/main.go

```diff
@@ -72,15 +77,36 @@ func main() {
 		logger.Fatalf("failed to start webhook configmap watcher: %v", err)
 	}
 
-	// TODO(n3wscott): Send the logger to the controller.
 	options := webhook.ControllerOptions{
-		ServiceName:      "eventing-webhook",
-		ServiceNamespace: system.Namespace,
-		Port:             443,
-		SecretName:       "eventing-webhook-certs",
-		WebhookName:      "webhook.eventing.knative.dev",
+		ServiceName:    "webhook",
```

Comment:
Changing the name of the webhook service is likely to mean that anyone with the existing webhook installed is going to run with two webhooks until they delete the old one by hand.

Put this in "release notes" -- steal the format from `knative/serving` if we don't have it.

Reply:
done, and made a PR to update our template: https://github.com/knative/eventing/pull/357